server/lib/routes/get-verify-email.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const got = require('got');
+
+const logger = require('mozlog')('server.get-verify-email');
+const config = require('../configuration');
+
+module.exports = function () {
+
+  return {
+    method: 'get',
+    path: '/verify_email/',
+    process: function (req, res, next) {
+      req.url = '/';
+      console.log('items', req.query);
+
+      let data = {
+        code: req.query.code,
+        uid: req.query.uid
```

Comment:
@divyabiyani we need to add validation to make sure these 2 items are present in the query request. If they are not then do not make the request below. You need a simpler version of this example: https://github.com/mozilla/fxa-content-server/blob/ef2e28d9a71e10ee226b107fef2f31f2d73c1223/server/lib/routes/post-csp.js#L19 (use the `joi` module to validate the structure of `uid` and `code`.  See the format of those 2 fields here: https://github.com/mozilla/fxa-auth-server/blob/7226ce0f627a0510ed93d8056663b1d6c7f42702/docs/api.md#post-v1recovery_emailverify_code